R analysis

rleigh-codelibre opened the pull request:
Initial logic for plotting the profile data with R.  Use ggplot2 for boxplots.  Sample output below (note: data is not final, and the Java performance should improve for pixeldata with the open PR):

[metadata-realtime.pdf](https://github.com/openmicroscopy/ome-files-performance/files/770951/metadata-realtime.pdf)
[metadata-realtime-withjace.pdf](https://github.com/openmicroscopy/ome-files-performance/files/770952/metadata-realtime-withjace.pdf)
[pixeldata-realtime.pdf](https://github.com/openmicroscopy/ome-files-performance/files/770953/pixeldata-realtime.pdf)
[pixeldata-realtime-withjace.pdf](https://github.com/openmicroscopy/ome-files-performance/files/770954/pixeldata-realtime-withjace.pdf)

rleigh-codelibre committed:
Initial data analysis with R

rleigh-codelibre committed:
R: Add variants without JACE

rleigh-codelibre committed:
R: Add Windows plots

rleigh-codelibre committed:
R: Plot datasets on same plot using facet_wrap

sbesson commented:
Latest results with all the current PRs opened: 
[results-13-02-2017.tar.gz](https://github.com/openmicroscopy/ome-files-performance/files/773556/results-13-02-2017.tar.gz) and
[archive.zip](https://github.com/openmicroscopy/ome-files-performance/files/773558/archive.zip)

Commit:
R: Add three figures

Commit:
R: Add exponents on scales

Commit:
R: Rescale to seconds

Commit:
R: Jace figure variants

sbesson committed:
Add benchmarking tabular results under version control

This should be the output of the OME-FILES-performance-win job and the Linux
benchmarking script using OME Files 0.3.0 and Bio-Format 5.3.3

Commit:
R: update plots

Commit:
R: update plots

Commit:
R: update plots

Commit:
R: Order datasets

Commit:
R: Improve scales

sbesson committed:
Update benchmark results

rleigh-codelibre committed:
Merge pull request #1 from sbesson/r-analysis

Update benchmark results

Commit:
R: Plot separate invocation and repetition in loop

Commit:
R: Correct log scale lower limit
